recipe/bld.bat

```diff
@@ -1,10 +1,9 @@
 :: Ensure our geos will be used.
 set GEOS_DIR=%LIBRARY_PREFIX%
 rmdir %SRC_DIR%\geos-3.3.3 /s /q || exit 1
```

Comment:
The `geos-3.3.3` folder does not exist anymore in the main `basemap` repo because the package delegates on whatever GEOS the user has available (for example, self-built or system-wide). Therefore, this line is not needed now.